src/plugins/console/public/lib/osd/osd.js

```diff
@@ -174,20 +173,19 @@ function loadApisFromJson(
 // like this, it looks like a minor security issue.
 export function setActiveApi(api) {
   if (!api) {
-    $.ajax({
-      url: '../api/console/api_server',
-      dataType: 'json', // disable automatic guessing
+    fetch('../api/console/api_server', {
+      method: 'GET',
       headers: {
         'osd-xsrf': 'opensearch-dashboards',
       },
-    }).then(
-      function (data) {
+    })
+      .then(function (response) {
+        response.json();
+      })
+      .then(function (data) {
         setActiveApi(loadApisFromJson(data));
-      },
-      function (jqXHR) {
-        console.log("failed to load API '" + api + "': " + jqXHR.responseText);
-      }
-    );
+      })
+      .catch((error) => console.log("failed to load API '" + api + "': " + error));
```

Comment:
style nit - prefer template literals to string concatenation
```suggestion
      .catch((error) => console.log(`failed to load API '${api}': ${error}`));
```